internal/openapi/utils.go

```diff
@@ -95,3 +96,10 @@ func getArgs(v *float64, t string) []any {
 func isRef(ref string) bool {
 	return ref != "" && strings.ContainsAny(ref, "#")
 }
+
+func getSchemaType(schema *openapi3.Schema) string {
+	if schema.Type != nil && len(*schema.Type) == 1 {
+		return (*schema.Type)[0] // Handle multiple types?
```

Comment:
Maybe we should do something like https://github.com/grafana/cog/blob/90dc057b38f3028d041b63a0e5ebc6c9d8e15ec5/internal/jsonschema/generator.go#L148-L149?

(not in the scope of this PR though, we should just create an issue for it)